cleanlab_cli/dataset/upload_helpers.py

```diff
@@ -433,3 +443,10 @@ def save_feedback(feedback, filename):
     progress(f"Writing issues to {filename}...")
     dump_json(filename, feedback)
     success("Saved.\n")
+
+
+def extract_float_value(column_value):
```

Comment:
Needs some docs to detail what cases this regex is able to extract float values from

Reply:
https://docs.python.org/3/library/re.html#simulating-scanf

This regex maps to the C library's scanf() for floating points (%e, %E, %f, %g)